app/assets/stylesheets/homepage.css.scss

```diff
@@ -38,9 +42,9 @@
     color: white;
     font-size: 1.45em;
     text-decoration: none;
-    height: 76px;
+    height: 75px;
     display: block;
-    padding: 12px 0px 0 12px;
+    padding: 9px 13px 0 13px;
```

Comment:
Line should be indented 2 spaces, but was indented 4 spaces<br>Shorthand form for property `padding` should be written more concisely as `9px 13px 0` instead of `9px 13px 0 13px`